packages/gitbook/src/components/DocumentView/OpenAPI/style.css

```
@@ -178,6 +178,10 @@
     @apply prose-sm mt-2 text-dark/10 dark:text-light/10;
 }
 
+.openapi-schema-pattern {
```

Let's group with `.openapi-schema-example`:

```suggestion
.openapi-schema-pattern, .openapi-schema-example {
```

Otherwise `@apply prose-sm` generates a good amount of CSS and I'm not sure how the css optimizer deals with it.